### DISCUSSION TITLE
Calendar allowed dates

### DISCUSSION THREAD

**Discussion by Ark21bit** [primefaces/community#1543] [PrimeVue]
Is there a way to set allowed Dates in the "calendar" component like here:https://vue3datepicker.com/props/calendar-configuration/#allowed-dates or any other workaround? Only this stops for use in the project

**Comment by tugcekucukoglu** [comment]
You could check `disabledDates` property. https://primevue.org/calendar/#api.calendar.props.disabledDates

**Reply by Ark21bit** [reply]
it doesn't really suit me, since the reverse disabledDates (!disabledDates) is needed. or is there a way to ask through it?
